docs/config/community-packs.rst

```diff
@@ -2,14 +2,25 @@
 Community Packs
 ===============
 
-As of `30/6/2022`, there currently are no community created config packs that exist for Terra v6,
-excluding the default `Overworld`_ pack.
+| This page contains a list of community packs for Terra.
+| To add your pack to this list, submit a PR to the `Docs Repository <https://github.com/PolyhedralDev/TerraDocs>`__
 
-This is expected as Terra v6 has not been released for long, and we have not built up sufficient
-documentation to provide developers with the means to develop new packs for it.
 
-If you wish to get more out of Terra, or tweak things to better fit your vision, we would recommend
-modifying the `Overworld`_ pack, which provides many preset configurations and variables to play
-around with. See the :doc:`/config/development/modifying-existing-pack` guide to get started.
+Overworld pack
+==========
```

Comment:
The header underline should contain the same amount of characters as the heading

Reply:
Sphinx warning for reference
```
/home/astrash/pro/terra/docs/docs/config/community-packs.rst:10: WARNING: Title underline too short.

Overworld pack
==========
```